bfvmm/include/hve/arch/intel_x64/vmexit/wrmsr.h

```diff
@@ -45,6 +45,12 @@
 #define EXPORT_HVE
 #endif
 
+// -----------------------------------------------------------------------------
+// Helpers
+// -----------------------------------------------------------------------------
+
+void emulate_wrmsr(::x64::msrs::field_type msr, ::x64::msrs::value_type val);
```

Comment:
Can we rename this or take it out of the public header? It does the exact opposite of what emulate means now

Reply:
Correct, this will be removed from the public header when I make the change as the "execute" function will be used instead. I need to leave it here for now for Boxy until I finish this API change.

Reply:
:+1: